src/librustdoc/lib.rs

```diff
@@ -43,7 +43,7 @@ extern crate rustc_trait_selection;
 extern crate rustc_typeck;
 extern crate test as testing;
 #[macro_use]
-extern crate log;
+extern crate tracing as log;
```

Comment:
Why not just change all `log` to `tracing` instead?

Reply:
This was less work :shrug: feel free to make a follow-up PR